tests/integration/bigquery_project/definitions/example_incremental.sqlx

```diff
@@ -1,14 +1,28 @@
 WITH example_data AS (
```

Comment:
i realize you haven't changed this, but IMO this is confusing - move this `WITH` block down below the `config` block?